cl/sentinel/sentinel.go

```diff
@@ -242,14 +243,18 @@ func New(
 		return nil, err
 	}
 
+	maxSubnetCount := max(
```

Comment:
I think this should be `cfg.NetworkConfig.AttestationSubnetCount` + `cfg.BeaconConfig.SyncCommitteeSubnetCount` + `cfg.BeaconConfig.MaxBlobsPerBlock`

Reply:
hmm, make sense